### PR TITLE
Post topics

### PR DESCRIPTION
**Pull request by mcass19**
I did not write new tests because I thought it was not necessary, although I'm not sure, so if you think I should add them, let me know.

### CODE REVIEW

**Comment by fchavat** [comment]
Dev QA:
- If I am in the `Post your topics` user interface
- and I type inside `To discuss` or `To discuss` input
- and I click outside the input

Expected Result:
- The entry is saved and a new input appears, so I can type a new topic

Actual Result:
- No entry is added to the actual list

**Comment by mcass19** [comment]
> Dev QA:
> 
> * If I am in the `Post your topics` user interface
> * and I type inside `To discuss` or `To discuss` input
> * and I click outside the input
> 
> Expected Result:
> 
> * The entry is saved and a new input appears, so I can type a new topic
> 
> Actual Result:
> 
> * No entry is added to the actual list

@fchavat sorry for not having notified this !

image

Maybe I can invite you to the Trello board.

File: app/views/retrospectives/create.js.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+$('#posting').html("<%=j render('form') %>");
```
Comment (fchavat) [inline app/views/retrospectives/create.js.erb]: The hole form is being reloaded every time a new topic is added. Maybe the mention/discuss topic can be a partial and you can [append](http://api.jquery.com/append/) it to a list of topics every time a new one is added? 
What do you think?

File: app/assets/stylesheets/retrospective.scss [hunk 2]
```diff
@@ -39,10 +39,39 @@
...
     display: none;
   }
 }
 
+.topics {
+  margin: auto;
+  padding: 10px;
+  border-style: solid;
+  border-color: #7c7c7c;
+  border-width: .7px;
```
[the remark this answers is not in the document]
Comment (mcass19) [reply app/assets/stylesheets/retrospective.scss]: Great ! I've changed that in 9426f86.

**Comment by mcass19** [comment]
@fchavat I decided to change the name of the method to a more behavioral name and not render the complete form every time, as you proposed.

In addition, I have decided to make more changes to have a better and more orderly solution:
- Separate the logic of 'discuss_topics' and 'mention_topics' in two different actions with a descriptive name for each one
- I did partial to add each new topic.
- I made the text field 'required' so that it does not send empty requests to the server

All these changes are at 130ba51.

**Pull request merged**